checks/contributors.go

```diff
@@ -39,7 +39,6 @@ func Contributors(c *checker.CheckRequest) checker.CheckResult {
 
 	companies := map[string]struct{}{}
 	for _, contrib := range contribs {
-		//nolint:nestif
```

Comment:
Why not refactor the code? Instead of nolint

Reply:
The code doesn't need refactoring since it already meets the linter's standards. That's why I removed the nolint.

Reply:
🤦 , Thanks!